### DISCUSSION TITLE
Save the value change but when it is null it doesn't save

### DISCUSSION THREAD

**Discussion by marcostaborda** [adonisjs/core#3084] [Help]
When creating a field that can be null, I persist and try to change the value that works, but when I try to return to null the lucid says persists, but when I try to visualize it leaves the old value.

if I do a check before setting the field, like `dhi: data.dhi || null` works. Since in the validator, it is optional.

**Comment by arthurcvm** [comment]
Hi @marcostaborda, in update if your don't send the field he do not update. In this case the best solution is send null from the client to api, or check in update if is undefined set to null, example:

```ts
farm.dhi = data.dhi ?? null
```
